app/views/users/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1> This is an user</h1>
+<div class="main-container">
+<%= render partial: 'shared/user', locals: { user: @user } %>
+        <div class="bio-container">
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+        </div>
+    
+        <%= render partial: 'shared/user_posts', locals: { posts: @user.posts, ren: 1 } %>
+    <button><%= link_to 'See all posts', user_posts_path(@user) %></button>
+</div>
```

Comment:
- [ ] [REQUIRED] Also, I will like to recommend that you make some adjustments to the user show page so that the design looks like the wireframe provided. Kindly justify the contents of the `div`s to center, position the `comments` and the `likes` to the bottom right where they belong, and centralize the `See all posts` button as well. Let's also not forget to separate the `div` for the User name and the User image as I have recommended in the partial earlier.

### Your Design
<img width="1135" alt="Screenshot 2023-06-14 at 23 49 05" src="https://github.com/sanieni6/BlogApp/assets/58692590/2a6c1bc3-b306-42f9-89d7-cfd320b1aa8e">

### Wireframe provided
<img width="489" alt="Screenshot 2023-06-15 at 00 04 00" src="https://github.com/sanieni6/BlogApp/assets/58692590/a9b4b70c-1c17-448e-be2f-b59a386e1cbb">

Reply:
well resolved! ✔️